src/features/locators/locators.slice.ts

```diff
@@ -57,7 +57,10 @@ const locatorsSlice = createSlice({
       const newValue = { ..._locator, locator: { ..._locator.locator }, message, type, name, isCustomName };
 
       if (fullXpath !== locator && robulaXpath !== locator) {
-        newValue.locator.customXpath = locator;
+        locatorType !== LocatorType.cssSelector
+          ? (newValue.locator.customXpath = locator)
+          : (newValue.locator.cssSelector = locator);
```

Comment:
please, use `if`. ternary looks confusing here

Reply:
or you can write smth like following:
```newValue.locator = locatorType !== LocatorType.cssSelector
          ? { ...newValue.locator, customXpath: locator }
          :  { ...newValue.locator, cssSelector: locator };
```

Reply:
fixed